extensions/DPLforum/DPLforum_body.php

```diff
@@ -296,7 +296,7 @@ function parse( &$input, Parser $parser ) {
 		} elseif ( ( $this->bAddAuthor || $this->bAddCreationDate ||
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1067)